docker/metricbeat/metricbeat.yml

```diff
@@ -0,0 +1,9 @@
+metricbeat.config.modules:
+  path: ${path.config}/modules.d/*.yml
+  reload.enabled: true
+
+processors:
+  - add_docker_metadata: ~
```

Comment:
Not sure if this is actually relevant, but it was included in the default configuration file and I decided to keep it in here since we run it inside Docker